package/example/ios/VisionCameraExample-Bridging-Header.h

```diff
@@ -1,8 +1,3 @@
 //
 //  Use this file to import your target's public headers that you would like to expose to Swift.
 //
-
-#if VISION_CAMERA_ENABLE_FRAME_PROCESSORS
```

Comment:
That flag was never set for the example project and because now example swift plugin uses `import VisionCamera`, I decided to remove the dead code